src/mpi/spawn/comm_disconnect.c

```diff
@@ -107,7 +107,7 @@ int MPI_Comm_disconnect(MPI_Comm * comm)
     {
         mpi_errno =
             MPIR_Err_create_code(mpi_errno, MPIR_ERR_RECOVERABLE, __func__, __LINE__, MPI_ERR_OTHER,
-                                 "**mpi_comm_disconnect", "**mpi_comm_disconnect %C", *comm);
+                                 "**mpi_comm_disconnect", "**mpi_comm_disconnect %p", *comm);
```

Comment:
>  Are reviewers expected to point out all the changes like this?

There are consistency differences between script generated versions and the original version. Without consistency, the script may have to add unmaintainable complexities to account for trivial ad-hoc differences. Reviewers are expected to apply their judgement on where it is okay and where it is incorrect.

> Isn't it %C since *comm is MPI_Comm (not a pointer)?

User may give a NULL/invalid pointer and it is unsafe to access the pointer in `fn_fail`. Here is a mistake that I should use `comm` instead of `*comm`. Good catch!

Reply:
Oh, this is the commits that is there just to make the commit able to compile. The change gets deleted as we regenerate them. I probably can restructure the commit so this commits can be omitted. Could you ignore this commit for now?